### PR TITLE
chore: correct NDJSON extension for sample ecommerce data

### PR DESCRIPTION
**Pull request by hainenber**
### Description
_Describe what this change achieves._
The sample ecommerce data file is not JSON per se but actually is `[NDJSON](https://github.com/ndjson/ndjson-spec?tab=readme-ov-file)` as suggested by the accompanying Content-Type header.

This PR is to remediate it.

### Issues Resolved
Closes #[_insert issue number_]
N/A

### Version
_List the OpenSearch version to which this PR applies, e.g. 2.14, 2.12--2.14, or all._
All

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 
N/A

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kolchfa-aws** [comment]
@peterzhuamazon Could you please review this PR?

**Comment by kolchfa-aws** [comment]
@mgodwan Could you please review this PR?

File: _getting-started/ingest-data.md [hunk 1]
```diff
@@ -50,32 +50,32 @@ Use the following steps to create a sample index and define field mappings for t
...
     ```
-    wget https://raw.githubusercontent.com/opensearch-project/documentation-website/{{site.opensearch_major_minor_version}}/assets/examples/ecommerce.json
+    wget https://raw.githubusercontent.com/opensearch-project/documentation-website/{{site.opensearch_major_minor_version}}/assets/examples/ecommerce.ndjson
     ```
     {% include copy.html %}
 
 1. Define the field mappings provided in the mapping file:
     ```bash
-    curl -H "Content-Type: application/x-ndjson" -X PUT "https://localhost:9200/ecommerce" -ku admin:<custom-admin-password> --data-binary "@ecommerce-field_mappings.json"
+    curl -H "Content-Type: application/json" -X PUT "https://localhost:9200/ecommerce" -ku admin:<custom-admin-password> --data-binary "@ecommerce-field_mappings.json"
```
Comment (mgodwan) [inline _getting-started/ingest-data.md:71]: Thanks for catching this!

**Comment by mgodwan** [comment]
@hainenber
Thanks for making the change to ensure the file's naming/content-type adhere to specification. I agree with the overall change.

For my understanding, this is mostly around representation, or did you also experience issues while following this guide due to mismatch of content-type and actual file spec?

**Review by mgodwan — APPROVED** [review]

**Comment by hainenber** [comment]
Thanks for the review @mgodwan 

It is only for representation. I've got no issue running the old cuủls.

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @hainenber! And thank you @mgodwan for the review. Ready to merge.

**Pull request merged**